packages/core/src/DdSdkReactNative.tsx

```diff
@@ -196,21 +197,52 @@ export class DdSdkReactNative {
         UserInfoSingleton.getInstance().setUserInfo(user);
     };
 
+    /**
+     * Sets the user information.
+     * @param id: A mandatory unique user identifier (relevant to your business domain).
+     * @param name: The user name or alias.
+     * @param email: The user email.
+     * @param extraInfo: Additional information.
+     * @returns a Promise.
+     */
+    static setUserInfo = async (userInfo: {
+        id: string;
+        name?: string;
+        email?: string;
+        extraInfo?: Record<string, unknown>;
```

Comment:
This userInfo type should become the new type used by the UserInfoSingleton once setUser gets deprecated.